src/solver/solve.jl

```diff
@@ -266,3 +265,36 @@ function check_conservation(sol)
 
     return (; energy_conservation, mass_conservation, water_conservation)
 end
+
+write_diagnostics_as_txt(simulation::AtmosSimulation) =
+    write_diagnostics_as_txt(
+        simulation.output_writers[1],
+        simulation.output_dir,
+    )
+write_diagnostics_as_txt(writer, output_dir) = nothing
+
+"""
+    write_diagnostics_as_txt(writer, output_dir)
+
+Write diagnostics in DictWriter to text files. This function is
+added because currently we do not support writing scalars to netcdf.
```

Comment:
Let's mention the specific diagnostics that are compatible with this

Reply:
Specific type of diagnostics (ie, they have to be 1-element vectors). Let's also add a link here to the issue in ClimaDiagnostics